core/src/test/java/org/apache/calcite/rex/RexProgramBuilderBase.java

```diff
@@ -756,8 +759,8 @@ protected RexNode vParam(String name, RelDataType type) {
    */
   protected RexNode vParam(String name, int arg, RelDataType type) {
     assertArgValue(arg);
-    RelDataType nonNullableType = typeFactory.createTypeWithNullability(type, false);
-    return rexBuilder.makeFieldAccess(getDynamicParam(nonNullableType, name), arg);
+    RelDataType nullableType = typeFactory.createTypeWithNullability(type, true);
```

Comment:
The logic of the function has also been changed.

So I think the commit message is not accurate enough.

Reply:
How about `Adjust nullableType  and java doc for RexProgramBuilderBase`?

Reply:
Good point, I'll update the commit info.